### PR TITLE
Reference added

### PR DESCRIPTION
**Pull request by praksharma**
I've added a reference to my paper.

### CODE REVIEW

File: docs/user/research.rst [hunk 1]
```diff
@@ -68,7 +68,7 @@ Here is a list of research papers that used DeepXDE. If you would like your pape
 
 PINN
 ----
-
+#. P. Sharma, L. Evans, M. Tindall, & P. Nithiarasu. `Stiff-PDEs and Physics-Informed Neural Networks <https://doi.org/10.1007/s11831-023-09890-4>`_. *Archives of Computational Methods in Engineering*, 2023.
```
Comment (lululxvi) [inline docs/user/research.rst:71]: Keep the empty line after `----`

Comment (lululxvi) [inline docs/user/research.rst:71]: "Physics-Informed Neural Networks" --> lower case

Comment (praksharma) [reply docs/user/research.rst]: done

**Comment by lululxvi** [comment]
Please also update the list of universities and research institutes.

**Comment by praksharma** [comment]
I've added the university and the research institute.

**Pull request merged**